### DISCUSSION TITLE
Merge Mod Lists

### DISCUSSION THREAD

**Discussion by Ap6661** [PrismLauncher/PrismLauncher#1800] [Ideas]
I believe it would be a useful feature to be able to merge mods into one instance. It would also be beneficial to be able to update an existing instance with the .mrpack feature. That way it would keep its current configurations but allow for batch installing additional mods.

**Comment by TheKodeToad** [comment]
you are looking for https://github.com/PrismLauncher/PrismLauncher/issues/new :P
